examples/react/react-native-app/project.json

```diff
@@ -0,0 +1,81 @@
+{
+    "name": "examples-react-react-native-app",
```

Comment:
should this not just be `examples-react-native-app`

Reply:
changed it to `example-react-native-app`